### PR TITLE
Docs(Dgraph): Update "More about Dgraph Zero" to reflect the output of the /state endpoint, and provide an edit pass

### PR DESCRIPTION
**Pull request by aaroncarey**
The current "More about Dgraph Zero" topic (published at https://dgraph.io/docs/deploy/dgraph-zero/#more-about-state-endpoint) has outdated output for the /state endpoint. While updating this, I also provided a writing / editing pass on this material.

Question for reviewers: Are the GET and POST endpoints correctly listed here? From reading this, it sounds like /assign and /removeNode are (or should be) POST endpoints, but they are listed in the doc as GET endpoints. I'm happy to confirm if I can get access to a cluster, or see the output of the help commands for `dgraph zero`

Fixes DGRAPH-2214

---
This change is [Reviewable](https://reviewable.io/reviews/dgraph-io/dgraph/6769)

 
 
 
 
 
 
 
 

Docs Preview: [Dgraph Preview](https://dgraph-9bbf08221d-104057.surge.sh)

### CODE REVIEW

**Comment by bucanero** [comment]
> 
> Question for reviewers: Are the GET and POST endpoints correctly listed here? From reading this, it sounds like /assign and /removeNode are (or should be) POST endpoints, but they are listed in the doc as GET endpoints. I'm happy to confirm if I can get access to a cluster, or see the output of the help commands for `dgraph zero`
> 

at least from the way they're calling the `assign` and `removeNode` endpoint, it's a `GET` call (sends all the parameters and values directly on the URL).  In any case, I'll let the devs confirm this question 😄

**Review by bucanero — CHANGES_REQUESTED** [review]
on a general note, I think we need to find a good template (or format structure) to present API endpoints, with all the variables, possible values, and so on. Let's discuss it in the docs team :smile: 

> Reviewed 1 of 1 files at r1.
> *[Reviewable](https://reviewable.io/reviews/dgraph-io/dgraph/6769#-:-MK76CfS-fFM8yTdYOM6:b80vjoc)* status: all files reviewed, 3 unresolved discussions (waiting on @aaroncarey, @danielmai, @MichaelJCompton, and @NamanJain8)

___
*[wiki/content/deploy/dgraph-zero.md, line 16 at r1](https://reviewable.io/reviews/dgraph-io/dgraph/6769#-MK73AWMC7WPvfuXaerz:-MK73AWMC7WPvfuXaes-:b-bldc2n) ([raw file](https://github.com/dgraph-io/dgraph/blob/a8378ee550953c459b768449034ba727e195d4df/wiki/content/deploy/dgraph-zero.md#L16)):*
> ```Markdown
> You can see the options available for `dgraph zero` using the following command:
> 
> `dgraph zero --help`
> ```

I'd use:

```bash
dgraph zero --help
```
___
*[wiki/content/deploy/dgraph-zero.md, line 26 at r1](https://reviewable.io/reviews/dgraph-io/dgraph/6769#-MK75XnEEB9HYC3Mxsns:-MK75XnEEB9HYC3Mxsnt:b-tsczbf) ([raw file](https://github.com/dgraph-io/dgraph/blob/a8378ee550953c459b768449034ba727e195d4df/wiki/content/deploy/dgraph-zero.md#L26)):*
> ```Markdown
> machines would serve group 2. Zero monitors the space occupied by predicates in
> each group and moves them around to rebalance the cluster.
> ```

I'd add a heading to this section: `## Endpoints`
___
*[wiki/content/deploy/dgraph-zero.md, line 61 at r1](https://reviewable.io/reviews/dgraph-io/dgraph/6769#-MK75lp-37QAmy0f4p-P:-MK75lp-37QAmy0f4p-Q:bu1964t) ([raw file](https://github.com/dgraph-io/dgraph/blob/a8378ee550953c459b768449034ba727e195d4df/wiki/content/deploy/dgraph-zero.md#L61)):*
> ```Markdown
> cluster by supplying it as part of the body.
> 
> ## More about the /state endpoint
> ```

this could be a 3rd level heading `### More about the...`

**Review by aaroncarey — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/dgraph-io/dgraph/6769)* status: 0 of 1 files reviewed, 3 unresolved discussions (waiting on @bucanero, @danielmai, @MichaelJCompton, and @NamanJain8)

___
*[wiki/content/deploy/dgraph-zero.md, line 16 at r1](https://reviewable.io/reviews/dgraph-io/dgraph/6769#-MK73AWMC7WPvfuXaerz:-MK7NcTID7NxnNj1_0oi:b-fz19ui) ([raw file](https://github.com/dgraph-io/dgraph/blob/a8378ee550953c459b768449034ba727e195d4df/wiki/content/deploy/dgraph-zero.md#L16)):*
<details><summary><i>Previously, bucanero (Damián Parrino) wrote…</i></summary><blockquote>

I'd use:

```bash
dgraph zero --help
```
</blockquote></details>

Interesting. I agree we should standardize on when we use so-called `code font`. Our style guide says "put code-related text in code font". In the past, I have also applied this to anything typed at the command line. 

What do you think about including this in the style guide? (i.e., "put code-related or command-line text in code font").
___
*[wiki/content/deploy/dgraph-zero.md, line 26 at r1](https://reviewable.io/reviews/dgraph-io/dgraph/6769#-MK75XnEEB9HYC3Mxsns:-MK7Q-_38Ed--Dwt-8HW:b-70y67c) ([raw file](https://github.com/dgraph-io/dgraph/blob/a8378ee550953c459b768449034ba727e195d4df/wiki/content/deploy/dgraph-zero.md#L26)):*
<details><summary><i>Previously, bucanero (Damián Parrino) wrote…</i></summary><blockquote>

I'd add a heading to this section: `## Endpoints`
</blockquote></details>

Nice idea! Done.
___
*[wiki/content/deploy/dgraph-zero.md, line 61 at r1](https://reviewable.io/reviews/dgraph-io/dgraph/6769#-MK75lp-37QAmy0f4p-P:-MK7PxA8C9MHkHSf578Z:bllaydm) ([raw file](https://github.com/dgraph-io/dgraph/blob/a8378ee550953c459b768449034ba727e195d4df/wiki/content/deploy/dgraph-zero.md#L61)):*
<details><summary><i>Previously, bucanero (Damián Parrino) wrote…</i></summary><blockquote>

this could be a 3rd level heading `### More about the...`
</blockquote></details>

Great catch! Done.

**Review by bucanero — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/dgraph-io/dgraph/6769)* status: 0 of 1 files reviewed, 1 unresolved discussion (waiting on @aaroncarey, @bucanero, @danielmai, @MichaelJCompton, and @NamanJain8)

___
*[wiki/content/deploy/dgraph-zero.md, line 16 at r1](https://reviewable.io/reviews/dgraph-io/dgraph/6769#-MK73AWMC7WPvfuXaerz:-MKBZB8Y8KXk0Zx6bebX:b-lpdg7n) ([raw file](https://github.com/dgraph-io/dgraph/blob/a8378ee550953c459b768449034ba727e195d4df/wiki/content/deploy/dgraph-zero.md#L16)):*
<details><summary><i>Previously, aaroncarey (Aaron Carey) wrote…</i></summary><blockquote>

Interesting. I agree we should standardize on when we use so-called `code font`. Our style guide says "put code-related text in code font". In the past, I have also applied this to anything typed at the command line. 

What do you think about including this in the style guide? (i.e., "put code-related or command-line text in code font").
</blockquote></details>

In this case, I was just referring to the triple Markdown code-tags ` ``` `, where you can also specify syntax highlighting. So for example in this case, since it's a bash command, I'd use ` ```bash ` to show the command line. (we already use it for GraphQL, ` ```graphql`)

File: wiki/content/deploy/dgraph-zero.md [hunk 1]
```diff
@@ -6,51 +6,64 @@ weight = 8
...
-remove a dead Zero or Dgraph Alpha node. To remove dead Zero nodes, pass
-`group=0` and the id of the Zero node.
+Dgraph Zero controls the Dgraph cluster, and stores information about it. It
+automatically moves data between different Dgraph Alpha instances based on the
+size of the data served by each Alpha instance.
+
+Before you can run `dgraph alpha`, you must run at least one `dgraph zero` node.
+You can see the options available for `dgraph zero` using the following command:
+
+`dgraph zero --help`
```
Comment (bucanero) [inline wiki/content/deploy/dgraph-zero.md:16]: This was my idea here (using triple code-tags):

```suggestion
```bash
dgraph zero --help
```

**Review by aaroncarey — COMMENTED** [review]
Thanks for the suggestions, Damian!

File: wiki/content/deploy/dgraph-zero.md [hunk 2]
```diff
@@ -6,51 +6,65 @@ weight = 8
...
+
+`--replicas` is the option that controls the replication factor, the number
+ of replicas per data shard, including the original shard. When a new Alpha
+joins the cluster, it is assigned to a group based on the replication factor.
+If the replication factor is set to `1`, then each Alpha node will serve a
+different group. If the replication factor is set to `2` and you launch four
+ Alpha nodes, then first two Alpha nodes would serve group 1 and next two
+machines would serve group 2. Zero monitors the space occupied by predicates in
+each group and moves them around to rebalance the cluster.
+
```
Comment (NamanJain8) [inline wiki/content/deploy/dgraph-zero.md:27]: We actually can't set the replication factor to 2. It throws this error.
```
ERROR: Number of replicas must be odd for consensus. Found: 2
```
If the replication factor is set to `3` and you launch six Alpha nodes, then first three Alpha nodes would serve group 1 and next three machines would serve group 2. Zero monitors the space occupied by predicates in each group and moves them around to rebalance the cluster. Note that the replication factor has to be odd for consensus.

File: wiki/content/deploy/dgraph-zero.md [hunk 3]
```diff
@@ -62,121 +76,90 @@ The `/state` endpoint of Dgraph Zero returns a JSON document of the current grou
...
+  "cid": "cdcb1edb-8c81-4557-af99-ebed2b383e3c",
   "license": {
+    "user": "",
     "maxNodes": "18446744073709551615",
-    "expiryTs": "1578704367",
+    "expiryTs": "1597232699",
     "enabled": true
   }
 }
 ```
```
Comment (NamanJain8) [inline wiki/content/deploy/dgraph-zero.md:118]: The full json looks like this. I didn't post the full json in the JIRA ticket. Sorry, my bad.
```json
{
  "counter": "22",
  "groups": {
    "1": {
      "members": {
        "1": {
          "id": "1",
          "groupId": 1,
          "addr": "alpha2:7082",
          "leader": true,
          "amDead": false,
          "lastUpdate": "1603350485",
          "clusterInfoOnly": false,
          "forceGroupId": false
        },
        "2": {
          "id": "2",
          "groupId": 1,
          "addr": "alpha1:7080",
          "leader": false,
          "amDead": false,
          "lastUpdate": "0",
          "clusterInfoOnly": false,
          "forceGroupId": false
        },
        "3": {
          "id": "3",
          "groupId": 1,
          "addr": "alpha3:7083",
          "leader": false,
          "amDead": false,
          "lastUpdate": "0",
          "clusterInfoOnly": false,
          "forceGroupId": false
        }
      },
      "tablets": {
        "dgraph.cors": {
          "groupId": 1,
          "predicate": "dgraph.cors",
          "force": false,
          "space": "0",
          "remove": false,
          "readOnly": false,
          "moveTs": "0"
        },
        "dgraph.graphql.schema": {
          "groupId": 1,
          "predicate": "dgraph.graphql.schema",
          "force": false,
          "space": "0",
          "remove": false,
          "readOnly": false,
          "moveTs": "0"
        },
        "dgraph.graphql.schema_created_at": {
          "groupId": 1,
          "predicate": "dgraph.graphql.schema_created_at",
          "force": false,
          "space": "0",
          "remove": false,
          "readOnly": false,
          "moveTs": "0"
        },
        "dgraph.graphql.schema_history": {
          "groupId": 1,
          "predicate": "dgraph.graphql.schema_history",
          "force": false,
          "space": "0",
          "remove": false,
          "readOnly": false,
          "moveTs": "0"
        },
        "dgraph.graphql.xid": {
          "groupId": 1,
          "predicate": "dgraph.graphql.xid",
          "force": false,
          "space": "0",
          "remove": false,
          "readOnly": false,
          "moveTs": "0"
        },
        "dgraph.type": {
          "groupId": 1,
          "predicate": "dgraph.type",
          "force": false,
          "space": "0",
          "remove": false,
          "readOnly": false,
          "moveTs": "0"
        }
      },
      "snapshotTs": "22",
      "checksum": "18099480229465877561"
    }
  },
  "zeros": {
    "1": {
      "id": "1",
      "groupId": 0,
      "addr": "zero1:5080",
      "leader": true,
      "amDead": false,
      "lastUpdate": "0",
      "clusterInfoOnly": false,
      "forceGroupId": false
    },
    "2": {
      "id": "2",
      "groupId": 0,
      "addr": "zero2:5082",
      "leader": false,
      "amDead": false,
      "lastUpdate": "0",
      "clusterInfoOnly": false,
      "forceGroupId": false
    },
    "3": {
      "id": "3",
      "groupId": 0,
      "addr": "zero3:5083",
      "leader": false,
      "amDead": false,
      "lastUpdate": "0",
      "clusterInfoOnly": false,
      "forceGroupId": false
    }
  },
  "maxLeaseId": "10000",
  "maxTxnTs": "10000",
  "maxRaftId": "3",
  "removed": [],
  "cid": "2571d268-b574-41fa-ae5e-a6f8da175d6d",
  "license": {
    "user": "",
    "maxNodes": "18446744073709551615",
    "expiryTs": "1605942487",
    "enabled": true
  }
}
```
This is for cluster with 3 alpha and 3 zeros. If this looks very big for the doc, we can figure out if we should show output for a smaller cluster or something else.

**Review by NamanJain8 — CHANGES_REQUESTED** [review]

**Review by aaroncarey — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/dgraph-io/dgraph/6769)* status: 0 of 1 files reviewed, 5 unresolved discussions (waiting on @bucanero, @danielmai, @MichaelJCompton, and @NamanJain8)

___
*[wiki/content/deploy/dgraph-zero.md, line 16 at r1](https://reviewable.io/reviews/dgraph-io/dgraph/6769#-MK73AWMC7WPvfuXaerz:-MKHURoW-wzT9SDexlTp:boyexb3) ([raw file](https://github.com/dgraph-io/dgraph/blob/a8378ee550953c459b768449034ba727e195d4df/wiki/content/deploy/dgraph-zero.md#L16)):*
<details><summary><i>Previously, bucanero (Damián Parrino) wrote…</i></summary><blockquote>

In this case, I was just referring to the triple Markdown code-tags ` ``` `, where you can also specify syntax highlighting. So for example in this case, since it's a bash command, I'd use ` ```bash ` to show the command line. (we already use it for GraphQL, ` ```graphql`)
</blockquote></details>

Done. Thanks!
___
*[wiki/content/deploy/dgraph-zero.md, line 16 at r2](https://reviewable.io/reviews/dgraph-io/dgraph/6769#-MK5usp5GE1mA9gygfAo-r2-16:-MKHUTU408eNViCG26UA:b-896fix) ([raw file](https://github.com/dgraph-io/dgraph/blob/78013fe5ed1011d787d66affe80429ce33a3958a/wiki/content/deploy/dgraph-zero.md#L16)):*
<details><summary><i>Previously, bucanero (Damián Parrino) wrote…</i></summary><blockquote>

This was my idea here (using triple code-tags):

```suggestion
```bash
dgraph zero --help
```
</blockquote></details>

Done.
___
*[wiki/content/deploy/dgraph-zero.md, line 18 at r3](https://reviewable.io/reviews/dgraph-io/dgraph/6769#-MK5usp5GE1mA9gygfAo-r3-18:-MKHUUCl4Fj_7NfHDnH4:b-dtkds3) ([raw file](https://github.com/dgraph-io/dgraph/blob/e51aa5e80d966a640d965089cc6e3e3c63a7befb/wiki/content/deploy/dgraph-zero.md#L18)):*
<details><summary><i>Previously, NamanJain8 (Naman Jain) wrote…</i></summary><blockquote>

We need to close the code section as well by using ```
</blockquote></details>

Done, thanks!
___
*[wiki/content/deploy/dgraph-zero.md, line 27 at r3](https://reviewable.io/reviews/dgraph-io/dgraph/6769#-MK5usp5GE1mA9gygfAo-r3-27:-MKHWUygA70Q1sr5sN5o:bypjsna) ([raw file](https://github.com/dgraph-io/dgraph/blob/e51aa5e80d966a640d965089cc6e3e3c63a7befb/wiki/content/deploy/dgraph-zero.md#L27)):*
<details><summary><i>Previously, NamanJain8 (Naman Jain) wrote…</i></summary><blockquote>

We actually can't set the replication factor to 2. It throws this error.
```
ERROR: Number of replicas must be odd for consensus. Found: 2
```
If the replication factor is set to `3` and you launch six Alpha nodes, then first three Alpha nodes would serve group 1 and next three machines would serve group 2. Zero monitors the space occupied by predicates in each group and moves them around to rebalance the cluster. Note that the replication factor has to be odd for consensus.
</blockquote></details>

Thanks for these extra details! I decided to add this example error message to the docs so that customers will find this page if they search on this error.
___
*[wiki/content/deploy/dgraph-zero.md, line 118 at r3](https://reviewable.io/reviews/dgraph-io/dgraph/6769#-MK5usp5GE1mA9gygfAo-r3-118:-MKHX5ta4yPsv9hIiG2r:b-eu8ju0) ([raw file](https://github.com/dgraph-io/dgraph/blob/e51aa5e80d966a640d965089cc6e3e3c63a7befb/wiki/content/deploy/dgraph-zero.md#L118)):*
<details><summary><i>Previously, NamanJain8 (Naman Jain) wrote…</i></summary><blockquote>

The full json looks like this. I didn't post the full json in the JIRA ticket. Sorry, my bad.
```json
{
  "counter": "22",
  "groups": {
    "1": {
      "members": {
        "1": {
          "id": "1",
          "groupId": 1,
          "addr": "alpha2:7082",
          "leader": true,
          "amDead": false,
          "lastUpdate": "1603350485",
          "clusterInfoOnly": false,
          "forceGroupId": false
        },
        "2": {
          "id": "2",
          "groupId": 1,
          "addr": "alpha1:7080",
          "leader": false,
          "amDead": false,
          "lastUpdate": "0",
          "clusterInfoOnly": false,
          "forceGroupId": false
        },
        "3": {
          "id": "3",
          "groupId": 1,
          "addr": "alpha3:7083",
          "leader": false,
          "amDead": false,
          "lastUpdate": "0",
          "clusterInfoOnly": false,
          "forceGroupId": false
        }
      },
      "tablets": {
        "dgraph.cors": {
          "groupId": 1,
          "predicate": "dgraph.cors",
          "force": false,
          "space": "0",
          "remove": false,
          "readOnly": false,
          "moveTs": "0"
        },
        "dgraph.graphql.schema": {
          "groupId": 1,
          "predicate": "dgraph.graphql.schema",
          "force": false,
          "space": "0",
          "remove": false,
          "readOnly": false,
          "moveTs": "0"
        },
        "dgraph.graphql.schema_created_at": {
          "groupId": 1,
          "predicate": "dgraph.graphql.schema_created_at",
          "force": false,
          "space": "0",
          "remove": false,
          "readOnly": false,
          "moveTs": "0"
        },
        "dgraph.graphql.schema_history": {
          "groupId": 1,
          "predicate": "dgraph.graphql.schema_history",
          "force": false,
          "space": "0",
          "remove": false,
          "readOnly": false,
          "moveTs": "0"
        },
        "dgraph.graphql.xid": {
          "groupId": 1,
          "predicate": "dgraph.graphql.xid",
          "force": false,
          "space": "0",
          "remove": false,
          "readOnly": false,
          "moveTs": "0"
        },
        "dgraph.type": {
          "groupId": 1,
          "predicate": "dgraph.type",
          "force": false,
          "space": "0",
          "remove": false,
          "readOnly": false,
          "moveTs": "0"
        }
      },
      "snapshotTs": "22",
      "checksum": "18099480229465877561"
    }
  },
  "zeros": {
    "1": {
      "id": "1",
      "groupId": 0,
      "addr": "zero1:5080",
      "leader": true,
      "amDead": false,
      "lastUpdate": "0",
      "clusterInfoOnly": false,
      "forceGroupId": false
    },
    "2": {
      "id": "2",
      "groupId": 0,
      "addr": "zero2:5082",
      "leader": false,
      "amDead": false,
      "lastUpdate": "0",
      "clusterInfoOnly": false,
      "forceGroupId": false
    },
    "3": {
      "id": "3",
      "groupId": 0,
      "addr": "zero3:5083",
      "leader": false,
      "amDead": false,
      "lastUpdate": "0",
      "clusterInfoOnly": false,
      "forceGroupId": false
    }
  },
  "maxLeaseId": "10000",
  "maxTxnTs": "10000",
  "maxRaftId": "3",
  "removed": [],
  "cid": "2571d268-b574-41fa-ae5e-a6f8da175d6d",
  "license": {
    "user": "",
    "maxNodes": "18446744073709551615",
    "expiryTs": "1605942487",
    "enabled": true
  }
}
```
This is for cluster with 3 alpha and 3 zeros. If this looks very big for the doc, we can figure out if we should show output for a smaller cluster or something else.
</blockquote></details>

This is great, thank you! No worries about not including the full text in the Jira ticket :) Our site is already set up to load long code snippets into an expandable window, so I see no reason not to include the whole thing :)

File: wiki/content/deploy/dgraph-zero.md [hunk 4]
```diff
@@ -6,51 +6,65 @@ weight = 8
...
+Dgraph Zero controls the Dgraph cluster, and stores information about it. It
+automatically moves data between different Dgraph Alpha instances based on the
+size of the data served by each Alpha instance.
+
+Before you can run `dgraph alpha`, you must run at least one `dgraph zero` node.
+You can see the options available for `dgraph zero` using the following command:
+
+```bash
+dgraph zero --help
+
```
[the remark this answers is not in the document]
Comment (aaroncarey) [reply wiki/content/deploy/dgraph-zero.md:28]: Done!

**Comment by aaroncarey** [comment]
Thanks for the input, Damian and Naman! It should be ready for another review at the following staging URL:

https://dgraph-322914c531-103744.surge.sh/deploy/dgraph-zero/#more-about-state-endpoint

**Review by bucanero — APPROVED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/dgraph-io/dgraph/6769#-:-MKHkJlZB4vInfY-fH7S:bnfp4nl)* status: 0 of 1 files reviewed, 5 unresolved discussions (waiting on @bucanero, @danielmai, @MichaelJCompton, and @NamanJain8)

File: wiki/content/deploy/dgraph-zero.md [hunk 5]
```diff
@@ -62,121 +83,201 @@ The `/state` endpoint of Dgraph Zero returns a JSON document of the current grou
...
-    - predicates
-        - dgraph.type
-        - counter.val
+  - Members (HTTP port number is shown after each node name):
+    - alpha2:7082, id: 1, leader
+    - alpha1:7080, id: 2
+    - alpha3:7083, id: 3
+    - zero1:5080, id: 1, leader
+    - zero2:5082, id: 2
+    - zero3:5083, id: 3
```
Comment (NamanJain8) [inline wiki/content/deploy/dgraph-zero.md:241]: Alphas belong to group 1 and Zeros belong to group 2. We can follow the same format as earlier.

**Review by NamanJain8 — APPROVED** [review]
Thanks @aaroncarey. Please just address the small comment I made.

**Review by aaroncarey — COMMENTED** [review]
> Reviewed 1 of 1 files at r5.
> *[Reviewable](https://reviewable.io/reviews/dgraph-io/dgraph/6769)* status: all files reviewed, 6 unresolved discussions (waiting on @bucanero, @danielmai, @MichaelJCompton, and @NamanJain8)

___
*[wiki/content/deploy/dgraph-zero.md, line 241 at r5](https://reviewable.io/reviews/dgraph-io/dgraph/6769#-MK5usp5GE1mA9gygfAo-r5-241:-MKLd3Pc7H59637Efbm1:b-82gzef) ([raw file](https://github.com/dgraph-io/dgraph/blob/74d86ef0c90cbf7cbf7e3e7a1564b64295431e11/wiki/content/deploy/dgraph-zero.md#L241)):*
<details><summary><i>Previously, NamanJain8 (Naman Jain) wrote…</i></summary><blockquote>

Alphas belong to group 1 and Zeros belong to **group 0**. 
</blockquote></details>

Done. Thanks for catching my error!

**Review by aaroncarey — COMMENTED** [review]
> Reviewed 1 of 1 files at r6.
> *[Reviewable](https://reviewable.io/reviews/dgraph-io/dgraph/6769)* status: all files reviewed, 6 unresolved discussions (waiting on @bucanero, @danielmai, @MichaelJCompton, and @NamanJain8)

**Pull request merged**